### ISSUE TITLE
threatStatus IUCN links to dead page

### ISSUE THREAD

**Issue by mnhn-paul** [gbif/portal-feedback#4395]
In the verbatim view of a checklist taxon record (eg. https://www.gbif.org/species/203120303/verbatim) there is a link associated to the IUCN threatStatus (http://iucn.org/terms/threatStatus). this is a "dead link" it currently only leads to a "Page not found
" page at IUCN.

maybe this could be fixed?

![image](https://user-images.githubusercontent.com/26623703/200531969-71ab92dc-e1fb-4788-bf68-a3350325436c.png)

![image](https://user-images.githubusercontent.com/26623703/200532184-1bba3328-295c-473c-b201-0c1509245057.png)

**Comment by MortenHofft** [comment]
It comes from this extension https://rs.gbif.org/extension/gbif/1.0/distribution_2022-02-02.xml
I'm not sure what is required to get it fixed. If it is IUCN who should fix their URL. Or it is the extension that should be updated. 
The website simply show the extension and the qualified names as is I believe.
@thomasstjerne perhaps you can say more?

**Comment by mnhn-paul** [comment]
Thanks @MortenHofft I did now also drop the IUCN team a message on their contact form, maybe its only temporarily not available on their site?

**Comment by thomasstjerne** [comment]
I think some of those URIs were originally minted without the intention to make them resolve. The IPT requires them to be URIs

**Comment by MattBlissett** [comment]
Yes, I'm not sure who invented it (quite possibly us) but there's no requirement for them to resolve to a web page or anything else.  The rs.gbif.org and rs.tdwg.org ones do, as it's convenient.

**Comment by mnhn-paul** [comment]
ok thanks for that information.
I find it however rather misleading to have them displayed as active links on the webpage.

**Comment by MortenHofft** [comment]
> I find it however rather misleading to have them displayed as active links on the webpage.

Indeed if it doesn't point anywhere. For the website part we just recognise it as a url and hence make it into a link tag.
It would be a shame to remove that everywhere, so the only other option is for us to have a list of urls to not link. A bit awkward, but possible.
